src/main/treeDataProviders/issuesTree/issuesTreeDataProvider.ts

```diff
@@ -262,16 +285,24 @@ export class IssuesTreeDataProvider implements vscode.TreeDataProvider<IssuesRoo
         checkCanceled();
         let graphSupported: boolean = await this._scanManager.validateGraphSupported();
         checkCanceled();
-        let status: { eosDone: boolean; graphDone: boolean } = { eosDone: false, graphDone: !graphSupported };
 
-        progressManager.startStep('🔎 Scanning for issues', graphSupported ? getNumberOfSupportedPackgeTypes() + descriptorsCount : 0);
+        // 1. Build workspace dependecy tree for all the descriptors
```

Comment:
```suggestion
        // 1. Build workspace dependency tree for all the descriptors
```
+ There is only "1". Where is "2"?